uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequest.php

```diff
@@ -120,7 +124,9 @@ private static function validateLeaveDatesDoesNotOverlapContracts($params) {
 
     if ($contract['count'] > 1) {
       throw new InvalidLeaveRequestException(
-        "The Leave request dates must not have dates in more than one contract period"
+        'The Leave request dates must not have dates in more than one contract period',
+        'leave_request_overlapping_multiple_contracts',
+        'to_date'
```

Comment:
to_date is a non required field. What if we get this error on a single-day LeaveRequest? I think it would be better to use the from_date as the field here